iina/MPVController.swift

```diff
@@ -342,8 +342,13 @@ class MPVController: NSObject {
     strArgs.insert(command.rawValue, at: 0)
     strArgs.append(nil)
     var cargs = strArgs.map { $0.flatMap { UnsafePointer<Int8>(strdup($0)) } }
+    defer {
```

Comment:
AFAIK, `defer` is a good practice for releasing unsafe resources. Because you can use it right after the alloctaion code, it's more noticeable & error-resistant.